www-apps/otrs/otrs-5.0.18.ebuild

```diff
@@ -0,0 +1,153 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit user systemd
+
+DESCRIPTION="OTRS is an Open source Ticket Request System"
+HOMEPAGE="https://otrs.org/"
+SRC_URI="https://ftp.otrs.org/pub/${PN}/${P}.tar.bz2"
+
+LICENSE="AGPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE="apache2 fastcgi +gd ldap mod_perl +mysql pdf postgres soap"
+SLOT="0"
+
+REQUIRED_USE="|| ( mysql postgres )"
+
+DEPEND="media-libs/libpng:0"
+
+RDEPEND="dev-perl/Apache-Reload
+	dev-perl/Archive-Zip
+	dev-perl/DBI
+	dev-perl/IO-Socket-SSL
+	dev-perl/JSON-XS
+	dev-perl/LWP-UserAgent-Determined
+	dev-perl/Mail-POP3Client
+	dev-perl/Mail-IMAPClient
+	>dev-perl/Net-DNS-0.60
+	dev-perl/Template-Toolkit
+	dev-perl/Text-CSV_XS
+	dev-perl/TimeDate
+	dev-perl/XML-Parser
+	dev-perl/YAML-LibYAML
+	apache2? (
+		mod_perl? (
+			www-servers/apache:2
+			=www-apache/libapreq2-2* www-apache/mod_perl
+		)
+		!fastcgi? ( !mod_perl? ( www-servers/apache:2[suexec] ) )
+	)
+	fastcgi? (
+		dev-perl/FCGI
+		virtual/httpd-fastcgi
+	)
+	!fastcgi? ( !apache2? ( virtual/httpd-cgi ) )
+	gd? (
+		dev-perl/GD
+		dev-perl/GDTextUtil
+		dev-perl/GDGraph
+	)
+	ldap? ( dev-perl/perl-ldap )
+	mysql? ( dev-perl/DBD-mysql )
+	postgres? ( dev-perl/DBD-Pg )
+	pdf? (
+		>=dev-perl/PDF-API2-0.73
+		virtual/perl-Compress-Raw-Zlib
+	)
+	soap? (
+		dev-perl/SOAP-Lite
+		!=dev-perl/SOAP-Lite-0.711
+		!=dev-perl/SOAP-Lite-0.712
+	)
+	"
+
+OTRS_HOME="${EROOT%/}/var/lib/otrs"
```

Comment:
Referencing `${EROOT}` in global scope isn't allowed (https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-12000011.1). Likewise, `${EROOT}` is only valid in `pkg_*` phases and not `src_*` phases. A simple fix would be to remove `${EROOT}` here, instead use `${EROOT%/}/${OTRS_HOME}` in `pkg_*` phases and `${OTRS_HOME}` in `src_*` phases when used with ebuild helpers (insinto etc.) or `${EPREFIX%/}/${OTRS_HOME}` otherwise. See https://devmanual.gentoo.org/ebuild-writing/variables/index.html#root for more info.